### PR TITLE
TEST: Workflow testing - Please ignore

### PR DESCRIPTION
**Pull request by kylewillmon**
~~Adds a formula for the Phylum CLI~~

This got a bit complicated. I am now using this PR for testing and will open new PRs once I have a working solution.

### CODE REVIEW

**Review by cd-work — APPROVED** [review]
What are upstream policies on adding new formulas? Is this something we could get upstreamed to brew or is that unlikely?

**Comment by kylewillmon** [comment]
This formula meets most of the requirements to be upstreamed. This version of the CLI still includes `phylum update`, but the fix for that has [been merged](https://github.com/phylum-dev/cli/pull/797) and will be included in the next release.

But they also require that a project "be known" and "be used" and they [don't like self-submitted formulae](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff) even going so far as to say:

> We frown on authors submitting their own work unless it is very popular.

Because of that and because it is quite simple to maintain our own tap, I decided to stick with this for now and we can submit the formula upstream later.

**Comment by cd-work** [comment]
> We frown on authors submitting their own work unless it is very popular.

Honestly this makes a lot of sense, so should be fine to keep it like this for now.

**Review by maxrake — APPROVED** [review]

**Review by andreaphylum — APPROVED** [review]

**Comment by kylewillmon** [comment]
I am using the PR to test out new workflows. Once I've got things working, I will create new PRs for the workflows and the new formula.

File: Formula/phylum.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  url "https://github.com/phylum-dev/cli/archive/refs/tags/v3.12.1.tar.gz"
+  sha256 "631ecb17a8100facd0b9e82e597c737cd105c0734408b9f68dec1aea99205e57"
+  license "GPL-3.0-or-later"
+  head "https://github.com/phylum-dev/cli.git", branch: "main"
+
+  bottle do
+    root_url "https://github.com/phylum-dev/homebrew-cli/releases/download/phylum-3.12.1"
+    sha256 cellar: :any_skip_relocation, monterey:     "fd03416447a582420387542e23a1c1a965c245e282a053c139e4ce22ed4d0337"
+    sha256                               x86_64_linux: "e0e316b6ca6cce71ddd8bc84bc576e741a552cf7636d72b09da454bc5f1bac6a"
+  end
```
Comment (kylewillmon) [inline Formula/phylum.rb:13]: 🎉

**Pull request closed without merging**